big-o-notation/big-o.js

```diff
@@ -24,8 +24,8 @@ function uniqueQuadratic(words) {
     i < words.length;               // 2 * n = O(2n) ~= 0(n)
     i++                             // 2 * n = O(2n) ~= 0(n)
   ) {
-    const word = words[i];          // 3 * n = O(3n) ~= 0(n)
-    let isUnique = true;            // 4 * n = O(4n) ~= 0(n)
+    const word = words[i];          // 1 * 1 = O(1)
+    let isUnique = true;            // 1 * 1 = O(1)
```

Comment:
These still execute `n` times, since they're in the outer loop. Check all of your other lines though, the number of simple operations are off in a lot of places. None have 6 in reality.